ui/client/components/graph/node-modal/editors/expression/Duration/PeriodEditor.tsx

```diff
@@ -20,31 +19,32 @@ type Props = {
   showValidation?: boolean,
   readOnly: boolean,
   isMarked: boolean,
+  components: Array<string>,
 }
 
 const SPEL_PERIOD_DECODE_REGEX = /^T\(java\.time\.Period\)\.parse\('(.*?)'\)$/
 const SPEL_PERIOD_SWITCHABLE_TO_REGEX = /^T\(java\.time\.Period\)\.parse\('P([0-9]{1,}Y)?([0-9]{1,}M)?([0-9]{1,}W)?([0-9]{1,}D)?'\)$/
 const SPEL_FORMATTED_PERIOD = (isoPeriod) => `T(java.time.Period).parse('${isoPeriod}')`
 const UNDEFINED_PERIOD = {
```

Comment:
The same case as in DURATION - NONE_DURATION.

Reply:
Done